spec/acts_as_replaceable_spec.rb

```diff
@@ -109,6 +119,12 @@
       Person.where(:first_name => 'John').count.should == 1
     end
 
+    it "should correctly detect difference between blank and nil values" do
```

Comment:
This was part of my attempt to fix https://github.com/culturecode/stolo_connect/issues/772 but it turns out the issue lay elsewhere.  I left the test in since it doesn't seem to hurt.